packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -48,6 +49,9 @@ async function updatePackageDependency(
       return;
     }
     if (!currentVersion) {
+      if (!install) {
```

Comment:
Make this check before getting the latestVersion. It's a network call that is unnecessary if we're not forcing the install.